driver/conv_driver.hpp

```diff
@@ -1944,9 +1953,11 @@ int ConvDriver<Tgpu, Tref>::FindBackwardData(int& ret_algo_count,
 template <typename Tgpu, typename Tref>
 int ConvDriver<Tgpu, Tref>::FindBackwardWeights(int& ret_algo_count,
                                                 int request_algo_count,
-                                                std::vector<miopenConvAlgoPerf_t>& perf_results)
+                                                std::vector<miopenConvAlgoPerf_t>& perf_results,
+                                                context_t ctx)
 {
     wrw_auxiliary.resume(wall_enabled);
+    ResizeWorkspaceDev(ctx, ws_sizeof_find_wrw);
```

Comment:
Is this missing func invocation at line no.1933 and 1960 the real cause of why wrong ws size was computed and passed causing the #238?

Reply:
I do not know what us the real cause of #238. But this fix is definitely required.